content/post/proficiency_v_creativity.md

```diff
@@ -0,0 +1,63 @@
+
+But as much as I value a robust data warehouse and [accurate reporting](https://www.locallyoptimistic.com/post/reporting-is-a-gateway-drug/), that isn’t what has kept me working with data over the years.
+
+## Creativity
+The creativity of analytics is why I show up to work each morning, and I think that’s true for many of us. We want to turn data into new knowledge and products - to learn from the past in order to shape the future. We want to explore and uncover new insights that suggest a path forward for the business. We want to predict who is going to churn instead of reporting on who churned last week. We want to create beautiful and signal-rich visualizations that allow our colleagues to immediately grasp relevant trends and patterns. And though I argued that the *outputs* of analytics engineering encourage proficiency just a few paragraphs ago, I think the *emergence* of analytics engineering as a discipline is also an act of creativity. Very smart folks are thinking outside the box to create tools and processes that apply engineering best practices and solutions to the problems analysts face.
+
+Leading for creativity means fostering systems that consistently churn out fresh ideas, even if many of those ideas don’t turn out to be successful. Empowerment is critical - providing support and tools to enable the team and letting them run with it. Creativity also requires a real focus on the team’s dynamic and sense of psychological safety. Teams should celebrate individuals who take initiative and generate new ideas, but not all those ideas will ultimately be good ones. Folks need to feel safe failing, suggesting something they aren’t quite sure is fully baked, and giving authentic feedback on someone else’s idea in an honest but respectful way.
+
+Tactically, enabling creativity requires time to work on less structured projects and a clear path to pursue ideas. Folks need time for explorations that may not always result in clear outcomes. Sometimes you spend half a day investigating something you think will be meaningful, and it’s just not - or weeks building a model only to find out that you just can’t predict the outcome with any useful level of accuracy. That time wasn’t wasted if you learn something from it.
```

Comment:
In addition to noting that 

> time wasn’t wasted if you learn something from it

(which I agree with) you could possibly make this more forceful by saying that failures are the _necessary_ output of uncertain processes -- good research labs have lots of failed experiments and we need to make sure that our research teams produce _good research_ and that means producing a lot of failures (the higher we're aiming, the more often we'll miss).

^^ that is not concise enough, but maybe you want to capture something like that here?

Reply:
Something I've always struggled with is providing my team a framework for how to think about failures. When do you stop going down the rabbit hole? How many failures are too many without a win? How do you ensure you are learning from your efforts?  Or better yet, how do you set yourself up to win even when the road you go down leads to failure (I start thinking about antifragility). Not necessary to expand on, but something I imagine many folks struggle with.